Tests/UnitTests/OfflineEntitlements/CustomerInfoOfflineEntitlementsTest.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerInfoOfflineEntitlementsTest.swift
+//
+//  Created by Nacho Soto on 3/21/23.
+
+import Foundation
+import Nimble
+import XCTest
+
+@testable import RevenueCat
+
+@available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
+class CustomerInfoOfflineEntitlementsTest: TestCase {
```

Comment:
@aboedo no point writing tests for just `EntitlementInfo` and also for this. We can just check the whole thing here.